pubspec.yaml

```diff
@@ -10,8 +10,8 @@ environment:
 dependencies:
   analyzer: ">=0.30.0 <=0.31.0"
   barback: ">=0.15.2 <=0.15.2+14"
-  built_redux: ">=6.1.0 <8.0.0"
-  built_value: ">=4.2.0 <6.0.0"
+  built_redux: ^7.4.1
+  built_value: ">=4.2.0 <5.2.0" # >=5.2.0 is Dart 2 SDK only
```

Comment:
To clarify, this is technically unnecessary, but done to improve solving speed in Dart 1's Pub, correct?

Reply:
Correct